test/unit/processing/siti-agri/transform-header.test.js

```diff
@@ -48,6 +48,20 @@ describe('Transform header', () => {
     })
   })
 
+  test('for SFI return undefined if paymentRequestNumber is NaN', async () => {
```

Comment:
I wonder if these tests for SFI should be moved above SFI Pilot?

Reply:
agree, updated.